### DISCUSSION TITLE
37,000 Games Still Herald

### DISCUSSION THREAD

**Discussion by cedricchi** [ValveSoftware/Dota2-Gameplay#21523] [Show and tell]
How is this possible? I guess Valve's 11-years ranked system can do this to us. Shame on you Valve! Shame on you!
https://hawk.live/posts/biggest-dota-2-fan-has-played-37000-matches-but-is-still-a-herald

![image](https://github.com/user-attachments/assets/0425e701-27e6-4743-ac5d-5e319f7f263f)

**Comment by jerry4422cherry** [comment]
It’s definitely wild to see someone put in that many hours and still be in Herald! It really highlights how complex Dota 2 can be, especially with its ranking system. Skill doesn’t always translate to rank, and sometimes matchmaking can be unforgiving. It’s a shame for dedicated players who want to see their efforts reflected in their rank. Valve definitely has some work to do with their ranking algorithms! [federal marketplace](https://www.health-insurancemarket.com)

**Reply by cedricchi** [reply]
it is a shame that Valve never opened its matchmaking algorithm since 2013
it is bizarre that Immortal is actually the worst rank with all kinds of noobs, feeders, trolls, jungle bots, boosters
it does not make any sense that each rank is actually the same in terms of supports don't buy sentries instead jungle farm; pos 1 jungle like a bot never smell lane creeps which give much more gold and exp than neutral creeps; pos 2 jungle and never gank, pos 3 jungle and never TP (doesn't even have or buy TP scroll which is 100 gold)
from my journey 3 years ago from Herald to Legend IT IS ALL EXACTLY THE SAME, no skill or understanding of the game difference whatsoever at all
worst part is, @jeffhill assured us that games are "BALANCED", but it is all STOMPs, not even the smallest chance for me to come back
it is rigged as fk
e.g. https://www.dotabuff.com/matches/7976974750
in this match I tried my best only to be laughed by my enemies, once they said in channel "are you bots"? I admit my team are worse than easy bots because easy bots dont feed as fast as my team
